academy/ibc/ibc-tooling.md

```diff
@@ -0,0 +1,214 @@
+
+While going through the overview, it is recommended to try out all there is to discover: just click around and see what happens.
+
+These types of tools help maintain an overview of the overall IBC network, but can also assist with things like relayer selection, as they provide an overview of essential metrics when it comes to relaying.
+
+## MapOfZones - Explore the Cosmos network
+
+[MapOfZones](https://mapofzones.com/) is a Cosmos network explorer.
+
+By default, the explorer shows you a visual overview of the IBC network for the last seven days:
+
+![MapOfZones](images/mapofzones1.png)
+
+You can also change the time being visualized by the MapOfZones in the lower right-hand corner of the display. You can chose between:
+
+* 24 hours
+* Seven days
+* 30 days
+
+The overview is dynamic and gives you a good feeling of the current activity in the overall network and between specific chains. You can also include testnet chains in the overview - you just have to enable it by activating `Show testnet` in the upper right-hand corner of the overview. The MapOfZones also lets you:
+
+* See the overview in full-screen mode
+* Zoom in and out
+* See the overview in 2D and 3D
+* Filter the zones you see by:
+	* Activity: you can select all, most active, or least active chains.
+	* Trendline: you can chose between uptrend and downtrend.
+
+All of which can be found on the right-hand side of the overview display.
+
+The individual chains are visualized by circle icons, sometimes including the chain's logo. Additionally, you can see connecting lines between the different chains. These lines represent transactions between chains.
+
+<HighlightBox type="note">
+
+In the overview display of the Cosmos network, the lines are colored to signal the main direction of the transactions. The coloring goes from red to green. The "redder" a line, the more transactions are mainly sends. The "greener" it becomes, the more transactions are mainly receives.
+
+</HighlightBox>
+
+When you hover over a specific chain with your mouse cursor, a small overview of data for that chain is displayed.
+
+![MapOfZones - hovering over a zone](images/mapofzones2.png)
+
+The information displayed includes:
+
+* The number of peers
+* The number of IBC transfers (for the selected time period)
+* The IBC volume (in USD)
+* The success rate (in %)
+* The IBC volume out (in USD)
+* The IBC volume in (in USD)
+* A button for more `Details`
+
+If you click on the `Details`, you are directed to an overview with more in-depth information about the chain selected:
+
+![Detailed list for the Cosmos Hub](images/detailscosmoszone.png)
+
+In the overview, you can find a list of all the chains the selected chain is connected to. When you click a specific chain, you can see the channels between the selected chain and another chain:
+
+![Channels between the Cosmos Hub and Osmosis](images/cosmososmasischannels.png)
```

Comment:
This image or the other one on mintscan might be good opportunity to reiterate that there are canonical channels for ICS-20 and that all other channels will have been created accidentally by (unexperienced) relayers hence have practically no txs

Reply:
Will add the information in a highlight box in line 73. Should fit very well. Thank you for the input.